content/04-building-and-testing/02-QA.md

```diff
@@ -64,3 +64,6 @@ Developer-driven focus management is programmatically setting keyboard focus aft
 * Mobile menus that take over the whole page should receive focus when opened, so users don’t have to tab multiple times
 * Content should not overlap, or force users to scroll horizontally
 * All content should still be available. Do not hide things from mobile users.
+
+<!-- Next Button -->
+<a href='../deploy-your-app/deploy-intro'><div class="next-button"><h5 class="next-text">Next: Deploy Intro</h5></div></a>
```

Comment:
What does this project use to build the static content page and does it support relative links like this?

Reply:
The build is directly forked from the 'design-system', so it's using fractal to turn the Markdown files into static pages. The relative linking does work, so it should still be effective on the live site

Reply:
@cehsu What exactly do you mean by a "broken link checker"?

Reply:
We use one on vets-website: https://www.npmjs.com/package/metalsmith-broken-link-checker